configure.ac

```diff
@@ -2709,7 +2711,37 @@ then
 	SAVE_CPPFLAGS="$CPPFLAGS"
 	CPPFLAGS="$LIBDPDK_CPPFLAGS $CPPFLAGS"
 	AC_CHECK_HEADERS([rte_config.h],
-		[with_libdpdk="yes"],
+		[
+      with_libdpdk="yes"
+      AC_RUN_IFELSE(
+        [
+          AC_LANG_PROGRAM(
+            [[
+              #include <rte_version.h>
+            ]],
+            [[
+              return (RTE_VER_YEAR & 0xFF)
+            ]]
+          )
+        ],
+        [rte_version_major=0],
+        [rte_version_major=$?]
+      )
+      AC_RUN_IFELSE(
```

Comment:
AC_COMPILE_IFELSE does only compilation, it doesn't run the AC_LANG_PROGRAM snippet. I need AC_RUN_IFELSE to extract RTE_VER_YEAR  and RTE_VER_MONTH values from rte_version.h  in runtime.